docs/handbook/image-file-formats.rst

```diff
@@ -1208,6 +1208,9 @@ The :py:meth:`~PIL.Image.Image.save` method can take the following keyword argum
 
     .. versionadded:: 8.4.0
 
+**bigtiff**
```

Comment:
I suggest adding an underscore to the public API for readability (and in the code):
```suggestion
**big_tiff**
```

Reply:
Ok, done.